h/annotation_sync.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+
+from h.celery import celery
+
+from h.api import presenters
+from h.api.search.index import index
+from h.api.search.index import delete
+
+__all__ = ('index_annotation',)
+
+
+@celery.task
+def index_annotation(action, annotation):
```

Comment:
It might be nicer to have two different tasks for indexing and deleting, rather than relying on `action`. It'll make stuff easier to test, too.